sql-plugin/src/main/scala/com/nvidia/spark/rapids/Plugin.scala

```diff
@@ -353,7 +353,7 @@ object RapidsPluginUtils extends Logging {
     val resourceName = "spark-rapids-extra-plugins"
     val classLoader = RapidsPluginUtils.getClass.getClassLoader
     val resourceUrls = classLoader.getResources(resourceName)
-    val resourceUrlArray = resourceUrls.asScala.toArray
+    val resourceUrlArray = resourceUrls.asScala.toArray.distinct
```

Comment:
Please add a warning listing duplicates. 

This looks like a deployment error to me when there are multiple identical  entries on the classpath that the user might want to fix.

Reply:
Yes, I added the warning clause.